app/models/task.rb

```diff
@@ -0,0 +1,6 @@
+class Task < ApplicationRecord
+  validates :title, presence: true
+  validates :description, presence: true
+  validates :due_date, presence: true
+  validates :done, presence: true
```

Comment:
you can do this in one line
 validates :title, :description, :due_date, :done, presence: true